src/relay/netorkErrors.js

```diff
@@ -0,0 +1,3 @@
+// @flow
```

Comment:
typo in filename

Reply:
there is no necessity to have it defined in separate file, it's overkill ==> relay/environment.js